godal.go

```diff
@@ -2889,7 +3024,24 @@ func (g *Geometry) GeoJSON(opts ...GeoJSONOption) (string, error) {
 	wkt := C.GoString(gjdata)
 	C.CPLFree(unsafe.Pointer(gjdata))
 	return wkt, nil
+}
 
+// GML returns the geometry in GML format.
+func (g *Geometry) GML(switches []string, opts ...GMLOption) (string, error) {
```

Comment:
Please document what `switches` can be used for. If they are optional, add the switches as a GMLOption (c.f. CreationOption used in many places, can this also be used here?)